patroni/dcs/kubernetes.py

```diff
@@ -41,8 +41,22 @@ def __init__(self, use_endpoints=False):
         self._request_timeout = None
         self._use_endpoints = use_endpoints
 
-    def set_timeout(self, timeout):
-        self._request_timeout = (1, timeout / 3.0)
+    def configure_timeouts(self, loop_wait, retry_timeout, ttl):
+        # Normally every loop_wait seconds we should have receive something from the socket.
+        # If we didn't received anything after the loop_wait + retry_timeout it is a time
+        # to start worrying (send keepalive messages). Finally, the connection should be
+        # considered as dead if we received nothing from the socket after the ttl seconds.
```

Comment:
Maybe shorten a bit here:
`# If not, start worrying (sending keepalive messages) and consider connection dead after ttl seconds.`

Reply:
The sentence you suggest doesn't precisely describe what and why we are doing.